noah/src/main/resources/population/crawler/population/population/items.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
```

Comment:
I can't comment on the empty file, is that `__init__.py` required? 
I'm sure we can remove the `*.pyc`.